### DISCUSSION TITLE
Call to the community : Freeplane on tablet ?

### DISCUSSION THREAD

**Discussion by bepolymathe** [freeplane/freeplane#966] [User testimonials]
Hello, 

It's been several years since I had the use of a tablet. A change in my professional activity will lead me to do more oral presentations with (obviously) Freeplane. There are less and less tables for speakers (at least in my field) and more and more round tables where a tablet is much more practical than a computer. 

Do you use Freeplane successfully and smoothly on a tablet? If so, what kind?

Thanks in advance.

**Comment by euu2021** [comment]
My sister has a touchscreen Windows laptop and some time ago I tested FP on it, and it worked fine. I didn't use it for much time, though. If you want, I can test some specific aspects of your interest.

I think FP will work just fine in those Flex 2 in 1 laptops. If you need an actual tablet, though, the Microsoft Surface is probably the best option, although more expensive.

**Reply by bepolymathe** [reply]
Hey, this is very nice. I plan to use the tablet to its fullest... but primarily to read my notes in map form during the speech. So folding and unfolding easily, and even see the images that would be integrated into the node...

If I wanted to share the map on a projector I would do it with the computer instead

**Reply by euu2021** [reply]
I tested again, and noticed that the navigation experience is not as good as I thought.


https://user-images.githubusercontent.com/77707706/215860121-7ae62e51-3a42-4be3-95ab-5b33584abf2e.mp4


-fold/unfold: works fine. If the clicking area was bigger, it would be better, though.
-scrolling: it doesn't have that inertia effect that usually happens when using FP with a touchpad. This makes scrolling difficult. At 00:27 in the video, I try to scroll quickly, but it goes slowly. The Map Overview saves the day, here.
-zooming: there is no zooming with two fingers. Probably related to #381 
-"right click": using two fingers doesn't work as right click. Maybe, it can be configured on Windows, like happens with touchpads?

So, the experience is not on par with apps optimized for touchscreen, but it is usable.

**Reply by bepolymathe** [reply]
All right. Thank you very much for the test @euu2021 it's very useful !
With the use in plan mode it allows to do things anyway 👍🏼

**Reply by euu2021** [reply]
now, I remembered that it's possible to change `Preferences…-Appearance-Size limits-Folding symbol width`. So, I think, changing it and the base distance between nodes can make the folding/unfolding work even better.

**Reply by bepolymathe** [reply]
Nice idea 👍🏼

**Reply by ArNicoS** [reply]
Hello everyone, sorry for asking about the basics: when you say you used FP on a tablet you mean android powered tablet? If so could you share the instructions to follow to have FP set up?

**Reply by ldmpub** [reply]
@ArNicoS sorry to disappoint you but there is yet no (known) way to run freeplane on an Android device.

Building an Android app is yet out of freeplane dev team ability (both for effort and skill reasons).

In the meantime community members are still searching for workarounds, please share yours if you have any...🤞

PS: Droidplane can display simple freeplane maps, the app is available in Google Play Store 👀

**Reply by bepolymathe** [reply]
It doesn't seem possible on Android but a bit more possible with a Microsoft surface...

**Reply by tino1003870** [reply]
On Android I am using "HalnaMind". No support of attributes, styles or these cool features, but ... it works.

**Reply by ldmpub** [reply]
@tino1003870: do you mean you can open Freeplane Mind Maps with HalnaMind? Or do you create new maps on Android with HalnaMind? 

Because as far as I remember (I performed tests some time ago) HalnaMind is a great (beside not maintained) Android app but can not open (nor edit nor save) native *.mm file = Freeplane files.

**Reply by yellek** [reply]
By default on my Microsoft Surface, dragging with my finger doesn't scroll the mind map. I'm hoping that this is just a setting somewhere because presenting from a Surface really needs this.

**Reply by Illegit** [reply]
I use normally the MS Surface pro7, and it works always: hope this check can help
![image](https://github.com/user-attachments/assets/b1285f5d-aea2-4497-9841-7cb737f81023)
but most important with the new FP releases you must install the FP something touchscreen, because of problem with java: actually I use FP 1.12.10_6 touchscreen.
